qe-tests/cypress/integration/models/plan.ts

```diff
@@ -179,4 +203,27 @@ export class Plan {
     this.run(name);
     this.waitForSuccess(name);
   }
+
+  restart(name: string): void {
+    cy.get(tdTag)
+      .contains(name)
+      .parent(tdTag)
+      .parent(trTag)
+      .within(() => {
+        clickByText(button, 'Restart');
```

Comment:
Ideally this traversal from the cell with the name to the row and down to the action cell shouldn't be necessary, if we have a good aria-label on that Restart button to indicate which plan it restarts. Making a note to possibly improve this in the future.